{"uuid":"db047833-e7b1-45f5-b434-e1dc8b1f1389","historyId":"9de1948df22177262de88da8dea8c06e","fullName":"user.TC39_VerifyThatTheUserCanRegisterSuccessfullyWithValidInformation.VerifyThatTheUserCanRegisterSuccessfullyWithValidInformation","labels":[{"name":"package","value":"user.TC39_VerifyThatTheUserCanRegisterSuccessfullyWithValidInformation"},{"name":"testClass","value":"user.TC39_VerifyThatTheUserCanRegisterSuccessfullyWithValidInformation"},{"name":"testMethod","value":"VerifyThatTheUserCanRegisterSuccessfullyWithValidInformation"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"at-project-template"},{"name":"subSuite","value":"user.TC39_VerifyThatTheUserCanRegisterSuccessfullyWithValidInformation"},{"name":"host","value":"AVEDN216L"},{"name":"thread","value":"10696@AVEDN216L.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"VerifyThatTheUserCanRegisterSuccessfullyWithValidInformation","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"The following asserts failed:\n\tForm still display expected [false] but found [true]","trace":"java.lang.AssertionError: The following asserts failed:\n\tForm still display expected [false] but found [true]\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)\r\n\tat user.TC39_VerifyThatTheUserCanRegisterSuccessfullyWithValidInformation.VerifyThatTheUserCanRegisterSuccessfullyWithValidInformation(TC39_VerifyThatTheUserCanRegisterSuccessfullyWithValidInformation.java:55)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1724567561796,"stop":1724567562344}